content/network/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Network of Research Software Quality
+description: Network Information
+identifier: Network
+weight: 10
+showDate: false
+groupByYear: false
+---
+
+## Network of Research Software Quality ##
+
+The EVERSE project is committed to improving the quality of software in European research in the international context. EVERSE will launch a Network of Research Software Quality to allow us to work with individuals and partner organisations to achieve this common goal.
+
+### Vision ###
+Our vision of the EVERSE Network of Research Software Quality is to establish a Community of Practice around improving the quality of research software in Europe and beyond. EVERSE and partners work to provide standard and documented practices around tools and training for software developers, researchers who code and research software engineers and service providers, to help them write better code. We help define the meaning and principles of software quality for the research domain. We work for enhanced recognition for software developers contributing to research activities.
```

Comment:
"around improving" -> "to improve"
"researchers who code and research software engineers and service providers" -> "researchers who code, research software engineers and service providers". I think it would be better to break the "to help them write better code" into a separate sentence as the list is so long. "This will help them...", e.g.